src/comment.cpp

```diff
@@ -304,7 +304,7 @@ void CommentParser::collectParameters(const char *fulltext, FileModule *root_mod
 
 		// Look for the group to which the given assignment belong
 		int i=0;
-		for (;i<groupList.size() && groupList[i].lineNo<firstLine;i++);
+		for (; (unsigned)i < groupList.size() && groupList[i].lineNo<firstLine;i++);
```

Comment:
I think its failing just because of given statement
`if (groupInfo.lineNo >= firstLine) `

It should be:
`if (groupInfo.lineNo < firstLine) `